napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -611,28 +319,41 @@ def addItem(
             lambda: self.handle_action(item, pkg_name, action_name)
         )
         widg.update_btn.clicked.connect(
-            lambda: self.handle_action(item, pkg_name, "install", update=True)
+            lambda: self.handle_action(
+                item, pkg_name, InstallerActions.install, update=True
+            )
         )
```

Comment:
In the past, we observed that connecting `lambda` to signals produced multiple problems in the past. 
It does not look like this place, but new contributors may treat this as a pattern in much more fragile place.